doc/tired_support.md

```diff
@@ -0,0 +1,197 @@
+# Tired Supported
+
+**Tier definitions**
+
+- **A — Production Ready:** fully tested and stable.
+- **B — Almost Production Ready:** partially tested; close to production.
+- **C — Experimental / Not Production Ready:** unfinished or volatile.
```

Comment:
what makes xtensa unfinished or volatile?

Reply:
Simply because we haven't received any positive news about Xtensa, I am open to assigning it a new level if there is some evidence. By the way, these are all initial values, and we are happy to adjust them based on feedback.

Reply:
enough for me. 

One more thing, should we provide or set up a full CI (testing) for tier A supported to ensure it is **fully tested and stable**?